post-scan-actions/azure-python-promote-or-quarantine/template.json

```diff
@@ -0,0 +1,344 @@
+            "subscriptionId": "[variables('scanResultTopicResourceSubscriptionID')]",
+            "properties": {
+                "mode": "Incremental",
+                "template": {
+                    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+                    "contentVersion": "1.0.0.0",
+                    "parameters": {},
+                    "variables": {},
+                    "resources": [
+                        {
+                            "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions",
+                            "apiVersion": "[variables('scanResultTopicAPIVersion')]",
+                            "name": "[format('{0}/{1}/{2}', variables('scanResultTopicNamespace'), variables('scanResultTopicName'), variables('scanResultTopicSubscriptionName'))]",
+                            "properties": {
+                                "defaultMessageTimeToLive": "PT1H",
+                                "maxDeliveryCount": 3,
+                                "lockDuration": "PT4M"
+                            }
+                        }
+                    ],
+                    "outputs": {
+                        "scanResultTopicSubscriptionResourceID": {
+                            "type": "string",
+                            "value": "[resourceId('Microsoft.ServiceBus/namespaces/topics/subscriptions', variables('scanResultTopicNamespace'), variables('scanResultTopicName'), variables('scanResultTopicSubscriptionName'))]"
+                        }
+                    }
+                },
+                "parameters": {}
+            }
+        },
+        {
+            "apiVersion": "2019-05-01",
+            "name": "[variables('scanningStoragePromoteQuarantineRoleAssignmentDeployment')]",
+            "type": "Microsoft.Resources/deployments",
+            "resourceGroup": "[variables('scanningStorageResourceGroupName')]",
+            "subscriptionId": "[variables('scanningStorageSubscriptionID')]",
+            "properties": {
+                "mode": "Incremental",
+                "template": {
+                    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+                    "contentVersion": "1.0.0.0",
+                    "parameters": {},
+                    "variables": {},
+                    "resources": [
+                        {
+                            "type": "Microsoft.Authorization/roleDefinitions",
+                            "apiVersion": "2018-07-01",
+                            "name": "[variables('scanningStoragePromteQuarantineRoleName')]",
+                            "properties": {
+                                "roleName": "[variables('scanningStoragePromteQuarantineRoleDefinitionName')]",
+                                "description": "The role for FSS promote or quarantine plugin to read blobs, delete blobs and generate SAS for blobs.",
```

Comment:
"description": "The role for FSS promote or quarantine plugin to read blobs, delete blobs, and generate SAS for blobs.",